nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -80,6 +80,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [imaginary](https://github.com/h2non/imaginary), a microservice for high-level image processing that Nextcloud can use to generate previews. Available as [services.imaginary](#opt-services.imaginary.enable).
 
+- [emptty](https://github.com/tvrzna/emptty), a dead-simple CLI display manager. Available as [services.xserver.displayManager.emptty](#options.html#ept-services.xserver.displayManager.emptty.enable)
```

Comment:
No this should go in 23.11

Reply:
@RaitoBezarius check most recent commit- that should be right.